infra/build/Makefile

```diff
@@ -32,7 +32,7 @@ KUSTOMIZE_VERSION := 3.6.1
 # Updated by Update Tooling Workflow
 KPT_VERSION := 1.0.0-beta.27
 # Updated by Update Tooling Workflow
-CFT_CLI_VERSION := 0.5.2
+CFT_CLI_VERSION := 0.9.0
```

Comment:
Hmm, looks like we have an issue with `update-tools`: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/actions/runs/4592192569/jobs/8109004037#step:3:73

Reply:
LoL - You already found it: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/1471

Reply:
Reverted